setup.py

```diff
@@ -0,0 +1,115 @@
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+"""Install and build injectdll distributions"""
+
+import os
+import shutil
+import subprocess
+from setuptools import setup
+from setuptools.dist import Distribution
+
+
+x86_cmake_arch_name, x64_cmake_arch_name = 'Win32', 'x64'
+x86_package_arch_name, x64_package_arch_name = 'x86', 'x64'
+
+arch_names_map = {
+    x86_cmake_arch_name: x86_package_arch_name,
+    x64_cmake_arch_name: x64_package_arch_name
+}
+
+build_dirname = 'build_'
+build_dll_dirs = ['./backends/dotnet/', './backends/hook/']
+package_dll_dirs = ['./injectdll/libs/dotnet/', './injectdll/libs/hook/']
+cmake_dirs = build_dll_dirs
+
+# cmake build for DLLs
+for arch in arch_names_map.keys():
+    for cmake_dir in cmake_dirs:
+        build_dir = cmake_dir + build_dirname + arch
+
+        os.makedirs(build_dir, exist_ok=True)
+
+        subprocess.check_call(['cmake', '-B ' + build_dir, '-S ' + cmake_dir, '-A ' + arch])
+        subprocess.check_call(['cmake', '--build', build_dir])
+
+# copy DLLs to the package
+for arch in arch_names_map.keys():
+    for build_dll_dir, package_dll_dir in zip(build_dll_dirs, package_dll_dirs):
+        for root, dirs, files in os.walk(build_dll_dir + build_dirname + arch):
+            for file in files:
+                if file.endswith('.dll'):
+                    src_file_path = os.path.join(root, file)
+                    dest_file_path = os.path.join(package_dll_dir + arch_names_map[arch], file)
+                    os.makedirs(os.path.dirname(dest_file_path), exist_ok=True)
+                    shutil.copy(src_file_path, dest_file_path)
+
+
+# mark the package is not pure Python code
+class BinaryDistribution(Distribution):
+    def is_pure(self):
+        return False
+
+
+setup(name='injectdll',
+      version='0.0.1',
+      description='A set of Python modules to inject DLls into applications for the Microsoft Windows',
+      keywords="windows gui .net inject testing test desktop dll wpf",
+      url="https://github.com/pywinauto/injected",
+      author='Mark Mc Mahon and Contributors',
```

Comment:
Mark didn't participate in injectdll development, but I need to carefully review the code one more time to check if there is some code written by him a long time ago.

Reply:
Suggest use "(C) 2018-2023 Vasily Ryabov and NNSU (Lobachevsky University) students" instead current version.

Reply:
It's for file headers. Of course `author` field should omit (C) and years.